Formula/a/aws-cdk.rb

```diff
@@ -16,7 +16,7 @@ class AwsCdk < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "e642ede7e4ff2d2c2628d400360a59ac3a013b6979224525fc3fd8114ed5d42e"
   end
 
-  depends_on "node"
+  depends_on "node@20"
```

Comment:
can we link to some upstream docs about the nodejs support policy?

Reply:
Hi @chenrui333 you can see the recommendation to use a LTS version of Node in the docs here, although it doesn't mention a version, I'll add it to the commit message

[Node dependency](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_prerequisites)

I have zero idea why the CI is blowing up for this tiny change.  `brew test` worked fine for me locally, but unfortunately I don't have bandwidth the debug this in docker :(

Reply:
it is just because of the commit style issue